src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/search/crosswikisearch/CrossWikiSearchPageObject.java

```diff
@@ -173,12 +180,19 @@ public void verifyResultsNumber(int number) {
   public void verifyNoPagination() {
     wait.forElementNotPresent(paginationContainerBy);
     PageObjectLogging.log("verifyNoPagination", "pagination is not visible on the page",
-                          true);
+        true);
+  }
+
+  public void verifyNoCommunitiesLink() {
+    wait.forElementNotPresent(otherCommunitiesLinkBy);
+    PageObjectLogging.log("verifyNoCommunitiesLink", "other communities link is not visible",
+        true);
   }
 
   public void verifyNoResultsCaption() {
     wait.forElementVisible(noResultsCaption);
-    Assertion.assertEquals(noResultsCaption.getText(), "No results found.");
+    String caption = noResultsCaption.getText();
+    Assertion.assertTrue(caption.equals("No results found.") || caption.contains("no matches"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "No results found." string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)